src/core/client/admin/components/ModerateCard/ModerateCardContainer.tsx

```diff
@@ -410,6 +410,7 @@ const ModerateCardContainer: FunctionComponent<Props> = ({
           moderationScopesEnabled={settings.multisite}
           userBanStatus={comment.author.status.ban}
           userRole={comment.author.role}
+          isMultisite={settings.multisite}
```

Comment:
Since we are already passing through `settings.multisite` for `moderationScopesEnabled`, it would be nice to combine these and just pass whether it is multisite through once but use it for both purposes.